gems/pending/spec/coverage_helper.rb

```diff
@@ -0,0 +1,12 @@
+require 'active_support/core_ext/kernel/reporting'
+
+# Require all ruby files for accuracte test coverage reports
+EXCLUSIONS_LIST = %w(/bin/ /spec/ /test/ appliance_console bundler_setup test.rb require_with_logging.rb VixDiskLibServer.rb)
+Dir.glob(File.join(GEMS_PENDING_ROOT, "**", "*.rb")).each do |file|
+  next if EXCLUSIONS_LIST.any? { |exclusion| file.include?(exclusion) }
+  puts "REQUIRING: #{file}"
```

Comment:
Debug.  Travis build times out.  It turns out there is a vendor directory in gems/pending on travis and the script required a file that started a rack server for the json gem.